central/convert/v1tostorage/auth_m2m_config_test.go

```diff
@@ -0,0 +1,41 @@
+package v1tostorage
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/pkg/testutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAuthM2MConfig(t *testing.T) {
+	config := &v1.AuthMachineToMachineConfig{}
+	require.NoError(t, testutils.FullInit(config, testutils.SimpleInitializer(), testutils.JSONFieldsFilter))
+
+	// Currently required as issuer won't be filled by FullInit.
+	config.IssuerConfig = &v1.AuthMachineToMachineConfig_GenericIssuerConfig{
+		GenericIssuerConfig: &v1.AuthMachineToMachineConfig_GenericIssuer{
+			Issuer: "https://stackrox.io",
+		},
+	}
+	storageConfig := AuthM2MConfig(config)
+
+	assertEqual(t, config, storageConfig)
+}
+
+func assertEqual(t *testing.T, a, b proto.Message) {
```

Comment:
Copy-paste alarm. You could eliminate copy-paste by merging both `_test.go` files in the parent directory. In this case, you won't have to have duplicate `assertEqual()` implementation.

Reply:
I found a place for the `assertEqual` method under `central/convert/testutils`, so other convert test cases may use it in the future.